### ISSUE TITLE
6.2.44: Login: wwsympa.fcgi: Error on incoming parameter "$data",

### ISSUE THREAD

**Issue by ikedas** [sympa-community/sympa#707]
Version
-------

6.2.44

Installation method
-------------------

Actual behavior
---------------

Running
`sympa.pl –upgrade –from=6.1.25 –to=6.2.44`

It crashes with this error message:
``` code
DIED: [Tue Jul 23 08:47:54 2019] wwsympa.fcgi: Error on incoming parameter "$data", it must be a ref on HASH or a ref on ARRAY at /home/sympa/bin/Sympa.pm line 422.
at /usr/share/perl5/CGI/Carp.pm line 362.
               CGI::Carp::realdie('[Tue Jul 23 08:47:54 2019] wwsympa.fcgi: Error on incoming pa...') called at /usr/share/perl5/CGI/Carp.pm line 461
                CGI::Carp::die('Error on incoming parameter "$data", it must be a ref on HASH...') called at /home/sympa/bin/Sympa.pm line 422
               Sympa::send_notify_to_listmaster('*', 'password_encrypted') called at /home/sympa/bin/Sympa/WWW/Auth.pm line 160
       Sympa::WWW::Auth::authentication('unifr.ch', '<loginName>', '<password>') called at /home/sympa/bin/Sympa/WWW/Auth.pm line 60
            Sympa::WWW::Auth::check_auth('unifr.ch', '<loginName>', '<password>') called at /home/sympa/bin/wwsympa.fcgi line 3155
                main::do_login() called at /home/sympa/bin/wwsympa.fcgi line 1544
```
 

Additional information
----------------------

See https://listes.renater.fr/sympa/arc/sympa-users/2019-07/msg00024.html

**Comment by racke** [comment]
Isn't that fixed by #706?

**Comment by ikedas** [comment]
@racke Thanks for fix!
